webapp/channels/src/packages/mattermost-redux/src/actions/admin.ts

```diff
@@ -627,7 +636,7 @@ export function getDataRetentionCustomPolicy(id: string): ActionFuncAsync<DataRe
     };
 }
 
-export function deleteDataRetentionCustomPolicy(id: string): ActionFuncAsync<{id: string}> {
+export function deleteDataRetentionCustomPolicy(id: string): ActionFuncAsync<{ id: string }> {
```

Comment:
eslint?

Reply:
Ideally, I think the preferred style would be to not have those extra space in type definitions similar to object literals, but no one's ever gone to add that to the ESLint config. I don't think it's set up to enforce whitespace consistently inside of type definitions in a number of cases

Reply:
I'll go add a ticket to figure out that rule. I'm sure there's other places where I've seen nitpicking over whitespace in those that we could avoid